src/Storages/Kafka/KafkaBlockInputStream.cpp

```diff
@@ -1,6 +1,5 @@
 #include <Storages/Kafka/KafkaBlockInputStream.h>
```

Comment:
Need to rename file?

Reply:
The same for `RabbitMQBlockInputStream.h`.

Reply:
Yes